tests/views/playingScreen-test.js

```diff
@@ -204,6 +204,50 @@ describe('PlayingScreen', function() {
     expect(clicked).toBe(true);
   });
 
+////********
+  it('creates a PlayingScreen, move video and checks touchEnd', function() {
```

Comment:
From an initial glance, this test does not appear to test the changes made in this PR. It looks like we should test that if ```handleVrPlayerMouseUp``` is called via a touchEnd event, we want to set the ```isVrMouseMove``` state to false. A state change is significant enough to add a unit test for.

Reply:
Unfortunately, I can't simulate touchMove event. It should to work by the logic, but it doesn't work in tests. May be, a do something wrong, may be it's just an bug or restriction of JEST.
I found same problems on stackoverflow but without resolving. 
So, that is the best testing way I found.

Reply:
Unfortunately, I can't simulate touchMove event. It should to work by the logic, but it doesn't work in tests. May be, a do something wrong, may be it's just an bug or restriction of JEST.
I found same problems on stackoverflow but without resolving. 
So, that is the best testing way I found.

Reply:
Ah, can you add a TODO on this then? Otherwise sounds good